src/app/product-list-page/components/product-list-page/filters/filters.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-filters',
+  templateUrl: './filters.component.html',
+  styleUrls: ['./filters.component.scss']
+})
+export class FiltersComponent {
+
+  public isChecked = false;
+  public rangeValue = 0;
```

Comment:
I don't see where `isChecked` and `rangeValue` params are used. Will it be implemented later?